Source/AtomicEditor/Editors/ResourceEditor.cpp

```diff
@@ -65,6 +65,7 @@ class EditorTabLayout: public TBLayout
                 }
                 else
                 {
+                    editor_->SendEvent("EditorResourceCloseCanceled");
```

Comment:
Yeah, unfortunately that shouldn't be defined like that either :/  

The problem with letting stuff like this slide is that there ends up being more and more cases of things like this, which leads to it being standard to do.  Let's not introduce more doing this...